lib/registerTypes/RegisterTypesLobbyPacks.h

```diff
@@ -54,6 +54,7 @@ void registerTypesLobbyPacks(Serializer &s)
 	s.template registerType<CLobbyPackToServer, LobbySetPlayer>();
 	s.template registerType<CLobbyPackToServer, LobbySetPlayerName>();
 	s.template registerType<CLobbyPackToServer, LobbySetTurnTime>();
+	s.template registerType<CLobbyPackToServer, LobbySetCheatAllowed>();
```

Comment:
please add to the very end (even if it won't be alphabetical)

Reply:
And if you are curious as to "why":
registerTypesXXX methods generate enumeration of types that can be (de)serialized.

Each type is serialized as (numeric type ID) (serialized type data)
So changing order will lead to a crash if game attempts to load type with wrong type ID